docs/figure_file_format.rst

```diff
@@ -0,0 +1,223 @@
+    }
+
+In general, OMERO.figure FileAnnotations are *not* linked to any Images because this would
+display them in the client UI alongside a delete button, allowing the user to accidentally delete
+the file, particularly since they may not recognise the file.
+
+JSON format
+-----------
+
+The JSON data format consists of a single 'Figure' object, with a list of 'panels'.
+There are a number of versions of the format, with minor differences between them. If you
+open an older version in the OMERO.figure app, it will handle the update and the file will
+be saved using the latest version.
+However, the figure export script may not handle older versions.
+
+This is an example of a minimal OMERO.figure JSON file::
+
+
+    {
+    // see older versions below
+    "version": 5,
+    "panels": [
+        {
+            // position of the panel on the page
+            "x": 200.0,
+            "y": 200.0,
+            "width": 150,
+            "height": 100,
+            // Z and T index
+            "theZ": 0,
+            "theT": 0,
+            // channel info & rendering settings
+            "channels": [
+                {
+                    "label": "528.0",
+                    "color": "00FF00",
+                    "inverted": false,
+                    "window": {
+                        "min": 0,
+                        "max": 1542.5,
+                        "start": 0,
+                        "end": 1542.5
+                    },
+                    "active": true
+                }
+            ],
+            // viewport percent zoom and offset from centre
+            "zoom": 150,
+            "dx": 16.96,
+            "dy": -22.42,
+            // labels on the panel
+            "labels": [
+                {
+                    "text": "GFP-INCENP",
+                    "size": 12,
+                    // positions are: top, left, right, leftvert, bottom, topleft, topright, bottomleft, bottomright
+                    "position": "top",
+                    "color": "00FF00"
+                },
+                {
+                    // 'live' timestamps, 'time' one of: secs, mins 
+                    "time": "milliseconds",
+                    "size": "12",
+                    "position": "topleft",
+                    "color": "FFFFFF"
+                }
+            ],
+            "rotation": 0,
+            // scalebar 
+            "scalebar": {
+                "show": true,
+                "length": 10,
+                "units": "MICROMETER",
+                "position": "bottomright",  // topright, topleft, bottomleft
+                "color": "FFFFFF",
+                "show_label": true,
+                "font_size": "14"
+            },
+            // See other shapes below
+            "shapes": [
+                {
+                    "type": "Rectangle",
+                    "x": 54.1,
+                    "y": 89.4,
+                    "width": 64.3,
+                    "height": 58.18,
```

Comment:
since we have a unit for scale bar. Unit for shape should be added too

Reply:
Below I say "Shapes on a panel use the Image coordinates". I don't see that changing. It works just the same as in OMERO itself.